src/main.c

```diff
@@ -67,6 +68,7 @@ uint32_t set_result_get_publicKey(void);
 #define OFFSET_CDATA 5
 
 #define MAX_RAW_TX 300
+#define MAX_RAW_MESSAGE 300
```

Comment:
maximum is here (so max 150 chars right?). Can it be pushed? If not update sha256 payload

Reply:
I cannot make sense of the above comment.

Anyway, wrt the maximum length of the message to be signed - I fixed the multi-chunk message signing and this means we can have a message of 1000s of bytes to sign [1]. But Ledger Nano S has the deficiency that it can't display messages longer than 132 bytes [2], so in the succeeding PR [3] the `MAX_RAW_MESSAGE` constant is set to 132.

[1] https://github.com/ArkEcosystem/ledger/pull/15/commits/d5f9b46bf2af30ab99f0459f2e9dbd1c7c2d1481
[2] https://github.com/ArkEcosystem/ledger/blob/046489cd44076ebd2ba47d1cf8207241b6086805/samples/sign-message.py#L54
[3] https://github.com/ArkEcosystem/ledger/pull/15